Fix S3-related tests

## What do?
The tagging and image upload tests continually fail, even though though there haven't been any changes to the underlying logic. For the time being I'll list the tests as no-ops and aim to fix them at a later point. If you have any ideas on how to fix this, @sea-kelp, I'd def welcome the expertise.

```console
=========================== short test summary info ============================
FAILED OpenOversight/tests/test_functional.py::test_image_classification_and_tagging
FAILED OpenOversight/tests/test_functional.py::test_anonymous_user_can_upload_image
============ 2 failed, 541 passed, 422 warnings in 61.78s (0:01:01) ============
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -43,7 +43,7 @@ services:
 
  postgres:
    restart: always
-   image: postgres:17
+   image: postgres:latest
    environment:
      POSTGRES_USER: openoversight
      POSTGRES_PASSWORD: terriblepassword

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -43,7 +43,7 @@ services:
 
  postgres:
    restart: always
-   image: postgres:latest
+   image: postgres:17
    environment:
      POSTGRES_USER: openoversight
      POSTGRES_PASSWORD: terriblepassword

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -35,7 +35,7 @@ services:
    entrypoint: >
      /bin/sh -c "
      sleep 5;
-     /usr/bin/mc config host add myminio http://minio:9000 minio ${MINIO_ROOT_PASSWORD};
+     /usr/bin/mc alias set myminio http://minio:9000 minio ${MINIO_ROOT_PASSWORD};
      /usr/bin/mc mb myminio/${S3_BUCKET_NAME};
      /usr/bin/mc anonymous set public myminio/${S3_BUCKET_NAME};
      exit 0;

